library/init/meta/uneval.lean

```diff
@@ -0,0 +1,74 @@
+/-
+Copyright (c) 2017 Microsoft Corporation. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Sebastian Ullrich
+-/
+prelude
+import init.meta.tactic
+
+open tactic
+
+meta class has_uneval (α : Type) :=
+(uneval : α → expr)
```

Comment:
Yes, it should really be `pexpr`. We do have `pexpr.to/of_raw_expr`, but it looked like a huge pain :/ .